README.rst

```diff
@@ -68,3 +68,28 @@ To re-enable them:
 
   config setprop janus-gateway DisabledPlugins ""
   signal-event nethserver-janus-update
+  
+Debug problems with Admin Api
+=============================
+
+The Janus Admin Api are used to debug problems (e.g. ssl, ice, ...).
+  
+``nethserver-janus`` automatically enables Admin Api on HTTP ``localhost`` on port ``7088``.
+
+To use it:
+
+1. make an ssh tunnel:
+
+.. code:: bash
+
+  ssh IP -L 7088:localhost:7088
+
+4. extract the Admin Api secret:
+
+.. code:: bash
+
+  grep admin_secret /opt/janus/etc/janus/janus.jcfg | cut -d ' ' -f 3
```

Comment:
Fix RST code block in nested list element:

```suggestion
4. extract the Admin Api secret: ::

     grep admin_secret /opt/janus/etc/janus/janus.jcfg | cut -d ' ' -f 3
```

Reply:
Same as above